h/static/scripts/permissions.coffee

```diff
@@ -37,10 +37,15 @@ module.exports = ['session', (session) ->
   # Typical use: annotation.permissions = permissions.private()
   ###
   private: ->
-    read: [session.state.userid]
-    update: [session.state.userid]
-    delete: [session.state.userid]
-    admin: [session.state.userid]
+    if not session?.state?.userid
```

Comment:
If the service depends on the session service, then we need to supply a session service mock when running the tests. I don't think adding defensive conditionals to the code just to support testing is a good idea.